src/server.py

```diff
@@ -470,6 +470,8 @@ def post(self):
                 respond_error(self, 400, 'Received invalid parameters')
                 return
 
+            model_helper.update_output_files_with_vars(config.output_files, execution_info.param_values)
```

Comment:
Please read my comment regarding moving the call to FinishListener and file_download_feature

Reply:
Yes no problem ;)